code/API_definitions/location-verification.yaml

```diff
@@ -489,17 +484,25 @@ components:
       content:
         application/json:
           schema:
-            $ref: "#/components/schemas/ErrorInfo"
+            allOf:
+              - $ref: "#/components/schemas/ErrorInfo"
+              - type: object
+                properties:
+                  status:
+                    enum:
+                      - 401
+                  code:
+                    enum:
+                      - UNAUTHENTICATED
+                      - AUTHENTICATION_REQUIRED
           examples:
             GENERIC_401_UNAUTHENTICATED:
-              summary: Generic Unauthenticated
               description: Request cannot be authenticated
               value:
                 status: 401
                 code: UNAUTHENTICATED
                 message: Request not authenticated due to missing, invalid, or expired credentials.
             GENERIC_401_AUTHENTICATION_REQUIRED:
-              summary: Generic Authentication Required
               description: New authentication is needed, authentication is no longer valid
```

Comment:
is it possible to explain the difference between "authentication is no longer valid" and "expired credentials" in more detail ?